pkg/drivers/outscale/utils.go

```diff
@@ -78,8 +81,10 @@ func (d *OscDriver) waitForState(vmId string, state string) error {
 
 func isThrottlingError(err error) bool {
 	if err != nil {
-		if strings.Contains(fmt.Sprint(err), "RequestLimitExceeded:") {
-			return true
+		for _, errorCode := range ThrottlingErrors {
+			if strings.Contains(fmt.Sprint(errorCode), fmt.Sprint(err)) {
```

Comment:
Using `fmt.Sprint(err)` to match http error code seems dangerous. Can we get the real http code instead ?

Reply:
Yes, changed to store these info in a custom structure